packages/p2p/src/components/buy-sell/row.jsx

```diff
@@ -40,13 +40,20 @@ export const RowComponent = React.memo(({ data, is_buy, setSelectedAd, style })
                     {data.display_price_rate} {data.transaction_currency}
                 </Table.Cell>
                 <Table.Cell>{data.display_payment_method}</Table.Cell>
-                {!is_agent ? (
+                {is_own_ad ? (
```

Comment:
```suggestion
                {!is_own_ad && (
                                    <Table.Cell>
                        <Button primary small onClick={() => setSelectedAd(data)}>
                        <Button
                            className='deriv-p2p__button--right-aligned'
                            primary
                            small
                            onClick={() => setSelectedAd(data)}
                        >
                            {is_buy ? localize('Buy') : localize('Sell')} {data.offer_currency}
                        </Button>
                    </Table.Cell>
                )}
```

Reply:
if we don't put a table cell in there, the bottom border will be missing.